src/constants/env.test.ts

```diff
@@ -0,0 +1,12 @@
+import { describe, expect, test } from 'vitest';
+import { Env } from './env';
+
+
+describe('./src/constants/env.ts', () => {
```

Comment:
Are these tests actually useful, considering the `test` environment is different from the environment the code actually runs in anyway?

Reply:
Ahhh, I understand. The environments that reload are the test ones, not the production ones. I thought about installing a package to validate these variables and guide the tests to ensure they contain a specific type of data.

The tests are not really important, but I did them anyway in case someone decides to refactor the test later with more precise logic